iStore/Constants.swift

```diff
@@ -21,6 +21,7 @@ let singleProduct = "https://dummyjson.com/products/"
 let limit = "/?limit=10&skip="
 
 let authenticationUrl = "https://dummyjson.com/auth/login"
+let currentUserURL = "https://dummyjson.com/auth/me"
```

Comment:
Why do you name it `currentUserURL`? it should be `userURL`. WDYT?

Reply:
I think both are ok.

Reply:
current user means for me that you have old user and you will have new user